src/features/mutation_minigame/nodes/mutation/mutation.gd

```diff
@@ -0,0 +1,28 @@
+extends "res://features/dynamicmenu/DragButton.gd"
+
+
+@export_enum("Insertion", "Deletion", "Substitution") var type: String = "Insertion"
+
+var mutation: Dictionary = {
+	"Insertion":{
+		"Icon": "insertion_icon.png",
+		"Tooltip_Text": "An insertion mutation adds a nucleotide, causing a frameshift."
+	},
+	"Deletion":{
+		"Icon": "deletion_icon.png",
+		"Tooltip_Text": "A deletion mutation deletes a nucleotide, causing a frameshift."
+	},
+	"Substitution":{
+		"Icon": "substitution_icon.png",
+		"Tooltip_Text": "blah blah blah"
+	},
+}
+
+func _ready():
+	
+	tooltip_text = mutation[type]["Tooltip_Text"]
```

Comment:
Is this being used at all?

Reply:
that statement is equivalent to saying `self.tootip_text = ......`. The way you can tell is because I never declare `tooltip_text`. So I am actually setting the tooltip text here.

Reply:
Ah, so it is! I didn't notice the inheritance.